components/UpdateCMSButton.tsx

```diff
@@ -0,0 +1,36 @@
+import { Button } from "@chakra-ui/react";
+import { Report, fetchReports } from "vd2hc";
+
+import { VD_REPORTS_ENDPOINT } from "./MintPage";
+import { ClaimData } from "./ReportPage";
+import { updateCMS } from "@/lib/updateCMS";
+
+export const UpdateCMSButton = ({ claims }: { claims: ClaimData[] }) => {
+  const updateCMSOnClick = async () => {
+    const item_id_array: string[] = [];
+    let cmsReports: Report[] = [];
+
+    let accessToken = process.env.NEXT_CMS_ACCESS_TOKEN as string;
```

Comment:
wrap in assertExists. Since it's the second use it might be cleaner to expose it as a constant in something like constants.ts